boards/arm/sam4l_ek/sam4l_ek.dts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <atmel/sam4lx4.dtsi>
+
+/ {
+	model = "Atmel SAM4L-EK Board with an Atmel SAM4LC4C SoC";
+	compatible = "atmel,sam4l_ek", "atmel,sam4lc4c", "atmel,sam4l";
```

Comment:
We do not need SoC compatibles, here.

Reply:
Sorry, it was not clear. Do you want I remove "atmel,sam4l" entry or all the sentence?

Reply:
You only need the "board" compatible here, not whatever refers to the SoC or SoC part number

Reply:
done.